tests/security/usbguard/usbguard.pm

```diff
@@ -20,6 +20,12 @@ use warnings;
 use testapi;
 use utils;
 
+sub restart_usbguard_allow_keyboard {
```

Comment:
Lili,

Can we consider it as a bug in aarch64 platform, lgtm from my side, but it should be better we can get confirmation from developer

Reply:
Good question. It is as expected because on aarch openQA environment there is a "keyboard" device and the default rule/policy is set to "block" so when usbguard service takes effect this device will be blocked.
It test cases I did some "rule removing" testings so after restart the usbguard will block "keyboard" too.